integration/findandmodify_compat_test.go

```diff
@@ -585,9 +585,8 @@ func TestFindAndModifyCompatUpsert(t *testing.T) {
 			command: bson.D{
 				{"query", bson.D{{"_id", bson.D{{"$exists", true}}}}},
 				{"upsert", true},
-				{"update", bson.D{{"_id", "replaced"}, {"v", "replaced"}}},
+				{"update", bson.D{{"_id", "int32"}, {"v", "replaced"}}},
```

Comment:
Hmm, to me this is a bit separate test case. In the previous test case ("replace") we were trying to set the value that didn't exist in any collection, just a new _id. In this test case, we try to set the value int32 which exists at least in some collections... I'd keep both of the test cases.

Reply:
I can keep both :+1: